scrapers/StateOfTexas.mjs

```diff
@@ -5,14 +5,20 @@ export default ({ database, DataScraper }) => {
 	return DataScraper(database, 'StateOfTexas', async () => {
 		const data = [];
 		const baseUrl = 'https://api.tnris.org/api/v1';
-		const id = '2679b514-bb7b-409f-97f3-ee3879f34448';
-		const collections = `collections/${id}`;
-		const resources = `resources?collection_id=${id}`;
-		const collectionsRequest = await fetch(`${baseUrl}/${collections}`);
-		const collection = await collectionsRequest.json();
-		const resourcesRequest = await fetch(`${baseUrl}/${resources}`);
-		const resource = await resourcesRequest.json();
+		const IdsRequest = await fetch(`${baseUrl}/collections_catalog/`);
+		const IdsJson = await IdsRequest.json();
+		const Ids = IdsJson.results.map(id => id.collection_id);
 
+		for(let i = 0; i < Ids.length ; i++) /*The classic for loop does not requires a big polyfill*/ {
+			
+
+		const collections = `collections/${Ids[i]}`;
+		const resources = `resources?collection_id=${Ids[i]}`;
+
+		const [collectionsRequest, resourcesRequest] = await Promise.all([fetch(`${baseUrl}/${collections}`), fetch(`${baseUrl}/${resources}`)]);
+
+		const [collection, resource] = await Promise.all([collectionsRequest.json(), resourcesRequest.json()]);
```

Comment:
I would prefer to see the fetch/json parsing happen in series instead of parallel.